w06-ruby/d1-dawn-peer-review/README.md

```diff
@@ -0,0 +1,44 @@
+# Project 1
+![request](imgs/request.gif)
+## Peer Review
+
+| Objectives |
+| :--- |
+| Create a Github Issue for your project |
+| Fork & then clone your partner's project |
+| Pair program to resolve the Github Issue & submit a Pull Request |
+
+**Please keep track of time -- you should spend roughly an hour on each project**
+
+###1. Create an Issue (10m)
+
+* Discuss what your application does at a high level. This should be one or two sentences, an elevator pitch. (5m)
+* Discuss 1 minor issue in your project. Make sure it's a simple issue, like a small addition, that could be solved relatively quickly with the tools we already have - jQuery, express, etc.
```

Comment:
what does this mean by "discuss"?  In writing?  with a partner?  If it's on the github issue then it should be combined with the next line to be more clear.